src/main/java/training/endava/skeleton/db/PersonRepository.java

```diff
@@ -0,0 +1,90 @@
+package training.endava.skeleton.db;
+
+
+import training.endava.skeleton.SkeletonRepository;
+
+import java.util.Iterator;
+import java.util.List;
+import java.util.Optional;
+
+public class PersonRepository implements SkeletonRepository<Person , Integer> {
+
+    //ArrayList<Person> persons = (ArrayList) MOCK_DB.getTable(Person.class);
```

Comment:
why is this commented ? you should save Person in MOCK_DB list not in the list of the repository